server/monitor/src/main/java/org/apache/accumulo/monitor/util/Table.java

```diff
@@ -110,7 +110,7 @@ public synchronized void generate(HttpServletRequest req, StringBuilder sb) {
       if (row.size() != columns.size())
         throw new RuntimeException("Each row must have the same number of columns");
 
-    boolean sortAscending = !"false".equals(BasicServlet.getCookieValue(req, "tableSort."
+    boolean sortDescending = "false".equals(BasicServlet.getCookieValue(req, "tableSort."
```

Comment:
I tested your changes on localhost using Uno and the sorting isn't working correctly for me.  I believe the solution is actually simpler than the changes you proposed.  You should keep the sortAscending variable (since the cookie and http request still use "asc") and only change the initial assingment.  I think this will work:

`boolean sortAscending = "true".equals(BasicServlet.getCookieValue(req, "tableSort."
        + BasicServlet.encode(page) + "." + BasicServlet.encode(tableName) + "." + "sortAsc"));`

This should work since the initial value of the cookie is null,the first pass will produce sortAscending = false. Then it should store the value correctly whatever it is from then on.